util/keyfile/src/mnemonic_acct.rs

```diff
@@ -1,3 +1,5 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
 // Copyright 2018-2022 The MobileCoin Foundation
```

Comment:
<!-- GRAPHITE_METADATA {"trueLocation":{"startLine":3,"endLine":3,"prLink":"https://app.graphite.dev/github/pr/mobilecoinfoundation/mobilecoin/2525"}} -->
<i class='graphite__hidden'>[Re: lines 3 to 3]<br/></i>

nit: remove<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/mobilecoin/2525?utm_source=unchanged-line-comment">Graphite</a>.</span>